modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -37,45 +37,45 @@ endif::[]
 .Procedure
 
 ifdef::migrating-3-4[]
-. Log in to `registry.redhat.io` with your Red Hat Customer Portal credentials:
+. Log in to `{mtc-registry}` with your Red Hat Customer Portal credentials:
 endif::[]
 ifdef::disconnected-3-4[]
-. On the workstation with unrestricted network access, log in to `registry.redhat.io` with your Red Hat Customer Portal credentials:
+. On the workstation with unrestricted network access, log in to `{mtc-registry}` with your Red Hat Customer Portal credentials:
```

Comment:
You can trim some of the duplication in the conditionalized text.

Reply:
What do you mean?